### ISSUE TITLE
Options to set resolution

### ISSUE THREAD

**Issue by vaibhavn1** [jeduan/cordova-plugin-crop#2]
Hi Mate,

Is there any way where we can resolution of the photo . like the width and the height?

Regards

Vaibhav

**Comment by Titoine** [comment]
+1
Only see quality, but cropping would need starting point (x,y),width and height.
Btw, could do it with canvas but It has poor performance.

**Comment by vyacheslavtyurikov** [comment]
+1
it would be great to be able to set the parameters: targetWidth and targetHeight

**Comment by jeduan** [comment]
Yep. That's currently hardcoded here and here in case you want to modify it or take a stab at a PR

https://github.com/jeduan/cordova-plugin-crop/blob/master/src/ios/CTCrop.m#L47
https://github.com/jeduan/cordova-plugin-crop/blob/master/src/android/CropPlugin.java#L39

**Comment by vyacheslavtyurikov** [comment]
It would be great to control it from JS. Could make such a possibility?

**Comment by Hesesses** [comment]
@jeduan are you planning to implement this?
